src/nebula-hooks/use-extended-theme.native.js

```diff
@@ -0,0 +1,9 @@
+import { useTheme } from '@nebula.js/stardust';
+
+const useExtendedTheme = () => {
```

Comment:
What is the reason that you are not using the non-native version?

Reply:
The index is off by one. The native one handles it by creating a proxy.